render.c

```diff
@@ -26,39 +27,49 @@ void render(struct slurp_output *output) {
 	cairo_paint(cairo);
 
 	struct slurp_seat *seat;
-	wl_list_for_each(seat, &state->seats, link) {
-		if (!seat->wl_pointer) continue;
-		if (!seat->has_selection) {
+	wl_list_for_each(seat, &state->seats, link)
+	{
+		struct slurp_selection *current_selection =
+			seat_get_current_selection(seat);
+		if (current_selection == NULL)
+			continue;
+		if (!seat->wl_pointer)
+			continue;
+		if (!current_selection->has_selection) {
 			continue;
 		}
 
-		if(!box_intersect(&output->logical_geometry, &seat->selection)) {
+		if (!box_intersect(&output->logical_geometry,
```

Comment:
Style: all lines below this one should be unchanged.

Reply:
I'm sorry, I don't  understand this one.

Reply:
Looking at the diff below, there are whitespace changes unrelated to this PR. It previously broke style, it doesn't anymore.